platforms/jvm/language-java/src/integTest/groovy/org/gradle/integtests/ExecIntegrationTest.groovy

```diff
@@ -910,6 +911,128 @@ class ExecIntegrationTest extends AbstractIntegrationSpec {
         "javaexec" | { File serverInfoFile -> javaExecSpecWithHttpServer(serverInfoFile) }
     }
 
+    def "execOperations.#method in #location resolves relative path correctly"() {
+        settingsFile << "include 'a'"
+        testDirectory.file("a").mkdirs()
+        testDirectory.file("a/build/test/folder").mkdirs()
```

Comment:
💅 There is a `createDir()` / `createDirs()` helpers

Reply:
TIL, thanks!